### ISSUE TITLE
acronyms and abbreviations broken?

### ISSUE THREAD

**Issue by jmtd** [philipdarke/simple-thesis#1]
Notation seems to be working (there is a notation chapter, and an entry, in the example document). But I cannot (yet) convince the glossary of terms/acronyms to appear.

Possibly the issue is PEBCAK; in either case, it would be good to put an example Acronym, and an example glossary entry, into the sample document, which would demonstrate exactly how to use them, and that they are working.

**Comment by philipdarke** [comment]
> Notation seems to be working (there is a notation chapter, and an entry, in the example document). But I cannot (yet) convince the glossary of terms/acronyms to appear.

The *Notation* and *Acronyms and abbreviations* sections are included when I run `make` on Ubuntu with `texlive-full` installed. How are you building the PDF?

> Possibly the issue is PEBCAK; in either case, it would be good to put an example Acronym, and an **example glossary entry**, into the sample document, which would demonstrate exactly how to use them, and that they are working.

I include example code in 2.2.6. Please take a look and let me know what you think is missing.

On the point in bold - the `glossaries` package is used to add the two sections but I do not support a traditional glossary. I don't think you would include one in a thesis?

**Comment by philipdarke** [comment]
I've tested on MacOS/MacTeX and it also builds, although there seems to be an Apple page rotation bug. I've seen this before when viewing PDFs on my phone.

**Comment by jmtd** [comment]
TL;DR: thanks to your reply and further investigation I have a plan for what to do and I don't propose any change to this repository to address my problems so I'll close this issue.

----

> The Notation and Acronyms and abbreviations sections are included when I run make on Ubuntu with texlive-full installed. How are you building the PDF?

Thanks for confirming. I've just built your main branch afresh and I get both sections. I must have cut something important in my private branch (where I'm layering my thesis content on top). I'll revisit to figure out what.

> texlive-full

I have not installed the full texlive on my system(s) (Debian). Pragmatically perhaps I should. The alternative would be recording texlive package dependencies here at a finer granularity, but perhaps that's busy-work, and we've both got actual theses to write! 

Side note, one such dependency is `fonts-lmodern`, or some other scalable font, without which the build fails with a fatal error. lacking `texlive-full`, I have to have `fonts-lmodern` *and* patch this in, to build:

```
diff --git a/simple-thesis.cls b/simple-thesis.cls
index 179addf..7227246 100644
--- a/simple-thesis.cls
+++ b/simple-thesis.cls
@@ -7,6 +7,8 @@
 \ProvidesClass{simple-thesis}[\filedate\space PhD thesis class]
 \typeout{See https://github.com/philipdarke/simple-thesis}
 
+\RequirePackage{lmodern}
+
 % Define 'oneside' option
 \RequirePackage{etoolbox}
 \newtoggle{oneside}
```
to avoid

    ! pdfTeX error (font expansion): auto expansion is only possible with scalable  fonts.

> On the point in bold - the glossaries package is used to add the two sections but I do not support a traditional glossary. I don't think you would include one in a thesis?

I'm not sure how common they are. I did a review of a half-dozen theses that I've stashed, but they are ones that were of interest to my area (or by people I know) so I can't claim it's a sample representative of a wider population. However, of the ones I looked at:

* Aaron: Nomenclature (albeit, maths symbols), pre-contents also Abbreviations
* Callaghan:'Notes and Abbreviations', pre-Chap 1 (adj. List of Figures etc);
        post ToC but not in ToC
* Jones: 'summary of notation' adj.Acks,not ToC
* Rhodes: Acronyms and Glossary in ToC, end, Refs/Acronyms/Glossary/Appendices     
* Suresh: Nomenclature, all symbols, in ToC, beginning
* Nothing in the university guidelines

My preference after that exercise was for there to be a section named "Glossary and Acronyms" that appeared in the ToC and was towards the beginning of the thesis (nearby lists of figures, etc.)

I think in my case I'll use the "Acronyms and Abbreviations" section as it exists in your template, and rename/repurpose it as a glossary.
